test-repos/nodeJS-v2/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejsv2",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "scripts": {
+    "swagger": "sls generate-swagger"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "serverless-auto-swagger": "file:../../../serverless-auto-swagger",
+    "serverless-offline": "^7.1.0"
```

Comment:
Shouldn't these also point to the root folder? Would be better to have `serverless-offline` installed at the root as a devDep once than for each of Node and Python serverless to install the same version separately